app/models/main.py

```diff
@@ -63,6 +64,9 @@ def __init__(self, astext_type=None):
         super(JSON, self).__init__(none_as_null=True, astext_type=astext_type)
 
 
+NestedMutable.associate_with(JSON)
```

Comment:
I think a comment here about why it's needed would be useful - something like `Enable tracking of nested updates/changes for all subsequent instances of the JSON class`.